clusterloader2/pkg/prometheus/manifests/prometheus-additional-scrape-configs.yaml

```diff
@@ -0,0 +1,12 @@
+{{$WINDOWS_NODE_NAME := DefaultParam .WINDOWS_NODE_NAME ""}}
```

Comment:
Maybe name this file "windows-additional-scrape-configs.yaml" instead?

Reply:
+1

Reply:
Seems all the prometheus yaml files have `prometheus-` prefix, so change it to "prometheus-windows-scrape-configs.yaml"